modules/custom/az_event/az_event_trellis/src/TrellisHelper.php

```diff
@@ -165,6 +169,10 @@ public function getEvents(array $trellis_ids) {
       catch (GuzzleException $e) {
       }
     }
+    // Make sure events are in Id order regardless of cached/fetched.
+    usort($events, function ($a, $b) {
+      return strcmp($a['Id'], $b['Id']);
+    });
```

Comment:
```suggestion
    // Make sure events are in Id order regardless of cached/fetched.
    // phpcs:disable Security.BadFunctions.CallbackFunctions.WarnCallbackFunctions
    usort($events, function ($a, $b) {
      return strcmp($a['Id'], $b['Id']);
   });
   // phpcs:enable Security.BadFunctions.CallbackFunctions.WarnCallbackFunctions
```
See https://app.probo.ci/#/dashboard/build/84716530-c7ac-48ac-94f1-982609a22b4e/task/637c1d48e0400002

Reply:
Or maybe just // phpcs:ignore on line 173?